versioned_docs/version-1.2/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Multiple Architecture Image Support
+sidebar_position: 110
+---
+
+# Multiple Architecture Image Support for ORAS
+
+ORAS can help you manage multiple architecture artifacts such as configurations, executables and container images.
+This example shows how you can use ORAS to manage executables for different operating systems and architectures.
+
+## Pushing the artifacts
+The first step is to push the artifacts to your container registry.
+
+First we push a Linux ARM64 executable:
+```shell
+% oras push --plain-http localhost:15000/oras:linux,arm64  --platform linux/arm64 --artifact-type 'application/vnd.oci.image.config.v1+json'  bin/linux/arm64/oras:application/octet-stream
```

Comment:
The flag name depends on the discussion of https://github.com/oras-project/oras/pull/1500

Reply:
changed to `--artifact-platform`